packages/uiweb/src/lib/components/space/SpaceBanner/SpaceBanner.tsx

```diff
@@ -1,15 +1,264 @@
+  title,
+  peopleInSpace,
+  participants,
+  time,
+  status,
+  host,
+  theme,
+  orientation,
+}) => {
   const { spaceBannerData, setSpaceBannerData } = useSpaceData();
 
   // Use spaceBannerData and setSpaceBannerData in your component
 
-  return <div>{/* Render your component content here */}</div>;
+  return (
+    <div>
+      <Container orientation={orientation}>
+        <ProfileContainer orientation={orientation}>
+          <PfpContainer>
+            <Pfp src="" alt="pfp" />
+          </PfpContainer>
+          <HostContainer>
+            <HostName>
+              
+              <Host>Host</Host>
+            </HostName>
+            <HostHandle></HostHandle>
+          </HostContainer>
+        </ProfileContainer>
+        {orientation === 'maximized' ? null : <Icon src="" />}
+        <Title orientation={orientation}>
+          {orientation === 'minimized' ? `${title?.slice(0, 30)}...` : title}
+        </Title>
+        <Status>
+          <Time orientation={orientation}>
+            <Icon />
+            <TimeText></TimeText>
+          </Time>
+          <Participants>
+            <ParticipantsIconContainer orientation={orientation}>
+              <ParticipantsIcon1 src="" alt="pfp"/>
+              <ParticipantsIcon2 src="" alt="pfp"/>
+              <ParticipantsIcon3 src="" alt="pfp" orientation={orientation} />
+            </ParticipantsIconContainer>
+            <ParticipantsText></ParticipantsText>
+          </Participants>
+        </Status>
+      </Container>
+    </div>
+  );
 };
+
+// Styling
+const Container = styled.div<{ orientation?: string }>`
+  display: flex;
+  flex-direction: ${(props) =>
+    props.orientation === 'maximized' ? 'column' : 'row'};
+  justify-content: ${(props) =>
+    props.orientation === 'maximized' ? 'space-between' : 'flex-start'};
+  align-items: ${(props) =>
+    props.orientation === 'maximized' ? 'flex-start' : 'center'};
+  padding: ${(props) =>
+    props.orientation === 'maximized' ? '16px' : '0 11px'};
+  gap: ${(props) => (props.orientation === 'maximized' ? '12px' : '8px')};
+  width: ${(props) =>
+    props.orientation === 'maximized' ? '709px' : '252.67px'};
+  height: ${(props) => (props.orientation === 'maximized' ? '200px' : '63px')};
+  background: linear-gradient(
+    87.17deg,
+    #ea4ee4 0%,
+    #d23cdf 0.01%,
+    #8b5cf6 100%
+  );
+  border-radius: ${(props) =>
+    props.orientation === 'maximized' ? '17px' : '24px'};
+  color: #f5f5f5;
+
+}`;
+
+const ProfileContainer = styled.div<{ orientation?: string }>`
+  display: ${(props) => (props.orientation === 'maximized' ? 'flex' : 'none')};
+  flex-direction: row;
+  justify-content: space-between;
+  width: fit-content;
+
+}`;
+
+const PfpContainer = styled.div`
+  margin: 5px;
+
+}`;
+
+const Pfp = styled.img`
+  height: 48px;
+  width: 48px;
+
+}`;
+
+const HostContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
```

Comment:
```suggestion
  align-items: center;
```

looks off in the screenshot you shared